db/migrations/003.do.change-sector-id-type.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE active_deals
+ALTER COLUMN sector_id TYPE BIGINT;
```

Comment:
Missing EOL at EOF.

(FWIW, I believe it's possible to configure Visual Studio Code always to add EOL at EOF.)

Reply:
Agreed with @NikolasHaimerl, either the Linter should catch it or we shouldn't discuss it in PRs

Reply:
Fair enough, I'll stop pointing out missing EOL at EOF.

Reply:
(I also added a note to https://github.com/space-meridian/roadmap/issues/166)